query_V1

```diff
@@ -58,30 +184,31 @@
 							JOIN ID_To_Artist j2 ON s.Song_ID = j2.Song_ID
 							JOIN Artist_PK ar ON ar.Artist_PK = j2.Artist_FK
 
-							WHERE 1
-							ORDER BY s.Title DESC, ar.Artist DESC");
+							WHERE s.title LIKE '%$searched%' or g.Genre LIKE '%$searched%' or ar.Artist LIKE '%$searched%' or a.Album LIKE '%$searched%' or s.Duration LIKE '%$searched%'
+							ORDER BY s.Title ASC, ar.Artist ASC");
 
 							//Runs and stores the query using the varibles $con (see nav.php) and $query
 							$result = mysqli_query($conn,$query);
 							//runs in a 'while' loop
 							while($output=mysqli_fetch_array($result))
 							{
+							
 						?>
 						<!--php is above. HTML is below. Used to output the query results-->
 
-						<div class="row" style="display: grid; grid-template-columns: 0.3fr 1.2fr 0.7fr 0.8fr 1fr 0.3fr; background-color: lightgrey;">
+						<div class="row" style="display: grid; grid-template-columns: 0.3fr 1.2fr 0.7fr 0.8fr 1fr 0.3fr; border-bottom: 0.1vw solid grey; transition: 0.15s ease-in-out;">
 
-							<div class="ID" style="grid-column: 1;"><p><?php echo $output['Song_ID']; ?></p></div>
+							<div class="ID" style="grid-column: 1;"><p style="margin: 0.5vw 2vw; font-size: 1vw;"><?php echo $output['Song_ID']; ?></p></div>
 
-							<div class="Title" style="grid-column: 2;"><p><?php echo $output['Title']; ?></p></div>
+							<div class="Title" style="grid-column: 2; border-right: 0.1vw solid grey;"><p style="margin: 0.5vw; font-weight: bold; font-size: 1vw;"><?php echo $output['Title']; ?></p></div>
```

Comment:
All of the changes in this section of code relate to the positioning of these row divs for ample space and visibility.